ocp_resources/observability_addon.py

```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ObservabilityAddon(NamespacedResource):
+    """
+    https://github.com/stolostron/multicluster-observability-operator/blob/main/docs/MultiClusterObservability-CRD.md
```

Comment:
is there an api doc?

Reply:
haven't found one yet; will update once I do